pkg/minishift/registration/utils.go

```diff
@@ -71,7 +71,7 @@ func RegisterHostVM(host *host.Host, param *RegistrationParameters) (bool, error
 	}
 
 	return doRegistrationHostAction(
-		"Registering machine using subscription-manager",
+		"-- Registering machine using subscription-manager ...",
```

Comment:
because of the progress dots?

Reply:
Maybe this should be on a new line anyways, with an indicative message.

because when I need to specify the username and password interactively, how is it shown now?

Reply:
@gbraad , Updated the PR now. It was not looking intutive :)

```
-- Registering machine using subscription-manager ...Red Hat Developers or Red Hat Subscription Management (RHSM) username: bgurung@redhat.com
Red Hat Developers or Red Hat Subscription Management (RHSM) password: [HIDDEN]
........... OK [33s]
```